### ISSUE TITLE
Cannot install Meteor through Chocolatey on Windows 7

### ISSUE THREAD

**Issue by dhulme** [meteor/meteor-chocolatey-installer#3]
I am unable to install Metoer 1.6 through chocolatey on Windows 7 64-bit. I have removed the old Meteor installer. Below is the output from Powershell. The first time I tried to install, I think I got an error about path being over 240 characters, but I have not been able to reproduce this since. I have attached the chocolatey log.

```powershell
PS C:\Windows\system32> choco install meteor
Chocolatey v0.10.8
Installing the following packages:
meteor
By installing you accept licenses for the packages.
Progress: Downloading meteor 0.0.1... 100%

meteor v0.0.1 [Approved]
meteor package files install completed. Performing other installation steps.
The package meteor wants to run 'chocolateyinstall.ps1'.
Note: If you don't run this script, the installation will fail.
Note: To confirm automatically next time, use '-y' or consider:
choco feature enable -n allowGlobalConfirmation
Do you want to run the script?([Y]es/[N]o/[P]rint): y

ERROR: A parameter cannot be found that matches parameter name 'file'.
The install of meteor was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\meteor\tools\chocolateyinstall.ps1'.
 See log for details.

Chocolatey installed 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - meteor (exited -1) - Error while running 'C:\ProgramData\chocolatey\lib\meteor\tools\chocolateyinstall.ps1'.
 See log for details.
```
[chocolatey.log](https://github.com/meteor/meteor-chocolatey-installer/files/1439327/chocolatey.log)

**Comment by romans127** [comment]
likewise

**Comment by ThaumRystra** [comment]
Likewise, got the 260 character error:

````
meteor package files install completed. Performing other installation steps.
The specified path, file name, or both are too long. The fully qualified file na
me must be less than 260 characters, and the directory name must be less than 24
8 characters.
ERROR: A parameter cannot be found that matches parameter name 'file'.
The install of meteor was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\meteor\tools\chocolateyinstal
l.ps1'.
 See log for details.

Chocolatey installed 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - meteor (exited -1) - Error while running 'C:\ProgramData\chocolatey\lib\meteo
r\tools\chocolateyinstall.ps1'.
 See log for details.
````

[chocolatey.log](https://github.com/meteor/meteor-chocolatey-installer/files/1440269/chocolatey.log)

**Comment by casey1111** [comment]
Same. Seeing "ERROR: A parameter cannot be found that matches parameter name 'file'."
In both Powershell and Admin-elevated Cmd.

**Comment by fallerd** [comment]
I was running this on a windows 7 machine and getting the same errors. I was able to solve and get meteor installed by updating Powershell to version 3, which is apparently for chocolatey to run correctly even though it doesn't give up any other outward errors. I just downloaded the update appropriate for my computer here:

https://www.microsoft.com/en-us/download/details.aspx?id=34595

**Comment by casey1111** [comment]
Thanks, @fallerd. Upgrading Powershell allowed for successful installation.

**Comment by dhulme** [comment]
I've started a pull request that should resolve this issue for those using Powershell 2

**Comment by abernix** [comment]
Thanks to @dhulme, the fix for this should be now be published in the _beta_ version of the new installer, [`meteor@0.0.2-beta-1`](https://chocolatey.org/packages/meteor/0.0.2-beta-1).  It may take up to 30 minutes to propagate through the Chocolatey CDN, but once that is ready, it should work via the following command, prior to the official 0.0.2 release (pending):

    choco install meteor --pre --version 0.0.2-beta-1

Please do help test!  It will take some time to get 0.0.2 approved by Chocolatey, so if there's anything else we want to release in 0.0.2 we should get that squared away soon.

**Comment by dhulme** [comment]
0.0.2-beta-1 confirmed working on Windows 7

**Comment by mdingena** [comment]
Tried `choco install meteor --pre --version 0.0.2-beta-1` as suggested above, and progressed slightly further, to stumble into this new error:

```
Downloading meteor 64 bit
  from 'https://packages.meteor.com/bootstrap-link?arch=os.windows.x86_64'
Progress: 100% - Completed download of C:\Users\Marc Dingena\AppData\Local\Temp\
chocolatey\meteor\0.0.2-beta-1\meteor-bootstrap-os.windows.x86_64.tar.gz (140.52
 MB).
Download of meteor-bootstrap-os.windows.x86_64.tar.gz (140.52 MB) completed.
Extracting C:\Users\Marc Dingena\AppData\Local\Temp\chocolatey\meteor\0.0.2-beta
-1\meteor-bootstrap-os.windows.x86_64.tar.gz to C:\Users\Marc Dingena\AppData\Lo
cal\Temp\chocolatey\meteor\0.0.2-beta-1...
ERROR: Exception calling "Start" with "0" argument(s): "The system cannot find t
he file specified"
The install of meteor was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\meteor\tools\chocolateyinstal
l.ps1'.
 See log for details.
```

**Comment by lacymorrow** [comment]
@abernix 's suggestion to use the current beta fixed this issue for me as well.

Windows7x64 via VMWare Fusion

> choco install meteor --pre --version 0.0.2-beta-1

**Comment by abernix** [comment]
@mdingena Could you either run the installer again with the `--debug` (and maybe `--verbose`) flag or post the log.  It should print the path to the log when the install exits, otherwise you might find it in `C:\ProgramData\chocolatey\logs`.  Either way, perhaps a new issue would be most appropriate since this one is said and done. :wink:

@lacymorrow So glad to hear it!  That's definitely the issue for Windows 7 users.  We'll submit it for official approval by Chocolatey soon, though I was hoping to get one more bug into it since they Chocolatey takes several days to approve installer updates.

**Comment by cforce** [comment]
The same issue is with kafka packge
